src/hooks/useModifiers/index.ts

```diff
@@ -0,0 +1,13 @@
+import {useMemo} from 'react';
+import * as Modifiers from '../../commons/modifiers';
+
+type Options = Parameters<typeof Modifiers.generateModifiersStyle>[0];
```

Comment:
Why not export `ModifiersOptions` from `Modifiers`?

Reply:
Fixed